### PR TITLE
adding new Bing Image Search quickstart sample

### PR DESCRIPTION
**Pull request by aahill**
## Purpose

* ...
This sample adds a new Bing Image Search code sample to compliment upcoming documentation changes on docs.microsoft.com

## Does this introduce a breaking change?

```
[ ] Yes
[ x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [https://github.com/aahill/cognitive-services-python-sdk-samples.git]
cd [cognitive-services-python-sdk-samples/samples/search/]
git checkout [new-quickstart]
run image-search-quickstart.py
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...
That the application outputs strings from a Bing Image Search query.

### CODE REVIEW

**Comment by aahill** [comment]
Hi @Imazuel, thanks for the response! 

I saw the file, but I need to create a new sample to support upcoming updates in the documentation. I've written a 5-minute quickstart that goes through this API sample, scoping it to a single user task to generate interest for more fully-featured docs,tutorials, and samples (like the one you mention).  Is that alright?

Cheers,
Aaron

**Comment by lmazuel** [comment]
@aahill As a customer using this repo, I think it's confusing :/
Could we discuss this by email with our PM @kirthik ?
Thank you

File: samples/search/image-search-quickstart.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from azure.cognitiveservices.search.imagesearch import ImageSearchAPI
+from azure.cognitiveservices.search.imagesearch.models import ImageType, ImageAspect, ImageInsightModule
```
Comment (lmazuel) [inline samples/search/image-search-quickstart.py]: Don't need this line at all

File: samples/search/image-search-quickstart.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+subscription_key = "Enter your key here"
+search_term = "canadian rockies"
+
+"""
+This application will search images on the web and print out first image result
+"""
+client = ImageSearchAPI(CognitiveServicesCredentials(subscription_key))
+
+try:
```
Comment (lmazuel) [inline samples/search/image-search-quickstart.py]: Remove the try/except, which is not required here

File: samples/search/image-search-quickstart.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+from azure.cognitiveservices.search.imagesearch import ImageSearchAPI
+from azure.cognitiveservices.search.imagesearch.models import ImageType, ImageAspect, ImageInsightModule
+from msrest.authentication import CognitiveServicesCredentials
+
+subscription_key = "Enter your key here"
+search_term = "canadian rockies"
```
Comment (lmazuel) [inline samples/search/image-search-quickstart.py]: remove this variable (not used)

**Pull request merged**